src/lib/internal-db/connections.ts

```diff
@@ -23,12 +23,12 @@ db.exec(`
   )
 `);
 
-db.exec(`
-  INSERT OR IGNORE INTO connections
-    (id, name, status, host, port, username, password, dc, nodes)
-  VALUES
-    (1, 'ScyllaDB Localhost', 'Offline', 'localhost', '9042', null, null, 'local', 1)
-`);
+// db.exec(`
+//   INSERT OR IGNORE INTO connections
+//     (id, name, status, host, port, username, password, dc, nodes)
+//   VALUES
+//     (1, 'ScyllaDB Localhost', 'Offline', 'localhost', '9042', null, null, 'local', 1)
+// `);
```

Comment:
Regarding that: this query only runs in right after the SQLite file is generated. Usually these credentials are default for LocalHost with ScyllaDB Docker Images, that's why I maintained it there.

It was conflicting with something else?

Reply:
This was causing the strange behavior on deleting that @Daniel-Boll had mentioned earlier because when you delete this from db, it just adds again when the file is re-executed.

We may keep it and find a way to avoid this specific entry from being deleted.